### PR TITLE
Update versionize on 1.3 release branch

### PR DESCRIPTION
**Pull request by roypat**
## Changes

Update versionize for 1.3  branch. Subsumes https://github.com/firecracker-microvm/firecracker/pull/3562

## Reason

backport

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] API changes follow the [Runbook for Firecracker API changes][2].
- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.
- [ ] New `TODO`s link to an issue.
- [ ] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1284,9 +1284,9 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
 
 [[package]]
 name = "versionize"
-version = "0.1.9"
+version = "0.1.10"
```
Comment (zulinx86) [inline Cargo.lock:1287]: Do we need to update versionize_derive as well?

Comment (roypat) [reply Cargo.lock:1287]: Nope, only the main versionize crate was updated

**Review by bchalios — COMMENTED** [review]
Are the other two commits cherry-picks from main? Should we mention that in the commit message? Don't remember what we have agreed

**Comment by roypat** [comment]
> Are the other two commits cherry-picks from main? Should we mention that in the commit message? Don't remember what we have agreed

They are not, I've written them from memory as the python framework has change significantly in the last few weeks 😅

**Review by bchalios — COMMENTED** [review]
I was wondering, why do we update Cargo.lock instead of Cargo.toml?

**Review by bchalios — APPROVED** [review]

**Pull request merged**